button-blinking-led/pru/Makefile

```diff
@@ -48,7 +48,7 @@ CFLAGS1 += -mmcu=am335x.pru1 -nostdlib -nodefaultlibs -nostartfiles
 
 # List of source files to compile for each PRU core.
 SRC0 := main0.c
-SRC1 := main1.S
+#SRC1 := main1.S
```

Comment:
Makefile would need more cleanup because I get an error:

```
$ make
pru-gcc -g -Os -Wall -Wextra -isystem include -minrt -mmcu=am335x.pru1 -nostdlib -nodefaultlibs -nostartfiles   -o out/pru-core1.elf
pru-gcc: fatal error: no input files
compilation terminated.
make: *** [Makefile:90: out/pru-core1.elf] Error 1
```